advanced/server-extension/README.md

```diff
@@ -81,60 +81,66 @@ communication with the server extension is contained in another file
 ```ts
 // src/index.ts#L6-L6
 
-import { requestAPI } from './server-extension';
+import { IFrame } from '@jupyterlab/apputils';
 ```
 
 In the `activate` function, the server extension is first called through
-a GET request on the endpoint _/hello/personal_. The response from the server
+a GET request on the endpoint _/jlab-ext-example/hello_. The response from the server
 is printed in the web browser console:
 
 ```ts
-// src/index.ts#L17-L23
+// src/index.ts#L33-L44
+
+palette: ICommandPalette,
+launcher: ILauncher | null
+=> {
+console.log('JupyterLab extension server-extension-example is activated!');
 
 // GET request
 try {
-  const data = await requestAPI<any>('personal');
+  const data = await requestAPI<any>('hello');
   console.log(data);
 } catch (reason) {
-  console.error(`Error on GET /hello/personal.\n${reason}`);
+  console.error(`Error on GET /jlab-ext-example/hello.\n${reason}`);
 }
 ```
 
 As the server response is not instantanious, the request is done asynchronously
 using the `await` keyword:
 
 ```ts
-// src/index.ts#L19-L19
+// src/index.ts#L40-L40
 
-const data = await requestAPI<any>('personal');
+const data = await requestAPI<any>('hello');
 ```
 
 To use that `await` keyword, the function must be declared as asynchronous
 using the `async` keyword:
 
 ```ts
-// src/index.ts#L14-L14
+// src/index.ts#L31-L31
 
-activate: async (app: JupyterFrontEnd) => {
+activate: async (
```

Comment:
Looks like we need the following line in this snippet?

Reply:
done.